src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -62,12 +155,45 @@ public void drive(Translation2d translation, double rotation, boolean fieldRelat
         for(SwerveModule mod : mSwerveMods){
             mod.setDesiredState(swerveModuleStates[mod.moduleNumber], isOpenLoop);
         }
-    }    
+    } 
+    
+    
+    public ChassisSpeeds getChassisSpeed() {
+
+        return swerveKinematics.toChassisSpeeds(getModuleStates());
+
+    }
+
+    public void setChassisSpeed(ChassisSpeeds chassisSpeed) {
+
+        SmartDashboard.putNumber("SetChassisSpeedX", chassisSpeed.vxMetersPerSecond);
```

Comment:
Go through and remove unneeded print statements and commented out code from path planner debugging